src/pages/partners/_premium_agent.module.scss

```diff
@@ -0,0 +1,85 @@
+@use 'features/styles/theme/theme-mixins' as *;
+@import 'features/styles/theme/variables';
+
+
+.wrap_container {
+  display: flex;
+  margin-bottom: 6rem;
+  gap: 0;
+  width: 90% !important;
+  margin-inline: auto;
+  justify-content: center;
+
+
+  @media (max-width: 1024px ) {
+    flex-wrap: wrap;
+
+  }
+
+  @media (max-width: 600px ) {
+    flex-wrap: wrap;
+    justify-content: start;
+
+  }
+
+    @include breakpoints (laptop) {
+      gap: 3rem;
+      padding: 40px 0 36px;
+    }
+
+
+  }
+
+  .wrapper {
+    width: 100%;
+    max-width: 56rem;
+  }
+
+  .premium-container {
+    @include breakpoints(tablet) {
+      padding: 40px 0 36px;
+    }
```

Comment:
Are this used somewhere?

Reply:
True, not needed now. Forgot to remove it. Thanks